data/strings/strings.txt

```diff
@@ -9843,6 +9843,17 @@
     zh-Hans = 下一页
     zh-Hant = 下一頁
 
+  [editor_time_from]
+    comment = Tab title in the time picker
+    tags = android
+    en = From
```

Comment:
Thanks! Can you please extract the correct From and To translations from these strings, generated by [this script](https://github.com/organicmaps/organicmaps/pull/5118/files#diff-f37cf5a5a9e99414057fcfa717fa9b85caf349abcfa3ebce55b91c25061d5e8a)?

```
en = From 6:00 to 11:00
ar = من الساعة 6:00 إلى 11:00
be = З 6.00 да 11.00
bg = От 6:00 до 11:00 ч.
ca = De 6:00 a 11:00
cs = Od 6:00 do 11:00
da = Fra 6:00 til 11:00
de = Von 6:00 bis 11:00
el = Από τις 6:00 έως τις 11:00
en-GB = From 6:00 to 11:00
es = De 6:00 a 11:00
et = Kell 6:00 kuni 11:00
eu = 6:00etatik 11:00etara
fa = از ساعت 6:00 الی 11:00
fi = Klo 6:00-11:00
fr = De 6h00 à 11h00
he = משעה 6:00 עד 11:00
hu = 6:00-tól 11:00-ig
id = Dari pukul 6:00 hingga 11:00
it = Dalle 6:00 alle 11:00
ja = 6:00から11:00まで
ko = 6:00~11:00
mr = 6:00 ते 11:00 पर्यंत
nb = Fra 6:00 til 11:00
nl = Van 6:00 tot 11:00
pl = Od 6:00 do 11:00
pt = Das 6:00 às 11:00
pt-BR = Das 6:00 às 11:00
ro = De la 6:00 la 11:00
ru = С 6:00 до 11:00
sk = Od 6:00 do 11:00
sv = Från 6:00 till 11:00
th = เวลา 6.00 - 11.00 น
tr = 6:00 - 11:00 arası
uk = З 6:00 до 11:00
vi = Từ 6:00 đến 11:00
zh-Hans = 从6:00到11:00
zh-Hant = 從 6:00 到 11:00
```